crates/core/src/vm.rs

```diff
@@ -532,11 +511,7 @@ impl ProgramVm for DbProgram<'_, '_> {
     }
 
     fn as_program_ref(&self) -> ProgramRef<'_> {
-        ProgramRef {
-            env: &self.env,
-            stats: &self.stats,
-            ctx: self.ctx(),
-        }
+        unreachable!()
```

Comment:
If it's unreachable, can we just remove it from the trait?

Reply:
@mamcx is `as_program_ref` safe to remove from the `ProgramVm` trait?

Reply:
Yes